specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/gallery.json

```diff
@@ -0,0 +1,1453 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SharedImageGalleryServiceClient",
+    "description": "Shared Image Gallery Service Client.",
+    "version": "2018-06-01"

```

Comment:
Should this version be 2018-06-01 or 2018-10-01

Reply:
gallery.json is not updated for 2018-10-01 since they are not using newer API version.. I removed that API model from newer version as its not changed..